web/src/lib/components/faces-page/people-card.svelte

```diff
@@ -17,6 +17,10 @@
     dispatch('change-name', person);
   };
 
+  const onSetBirthDateClicked = () => {
+    dispatch('set-birth-date', person);
```

Comment:
I realize this is following exist code in this component, but in general if the component doesn't do anything besides emit an event and it only does it in one place I usually inline it. All of these events could be inlined. Also, we should use typed events here as well. Basically just adding

```typescript
let dispatch = createEventDispatcher<{
  'change-name': PersonResponseDto,
  'set-birth-date': PersonResponseDto,
  'merge-faces': PersonResponseDto,
}>();
```

In all reality, these could/should probably all be `event: void,`, since the dto isn't changed and the parent is passing it in already.

Reply:
I changed all of them following your suggestion. However I left PersonResponseDto because it would need more refactorings.